src/components/LeftNavigation/LeftNavigation.tsx

```diff
@@ -78,7 +78,10 @@ export class LeftNavigation extends CommonComponent<ILeftNavigationProps, any> {
 
     private getSelectedId(options: Array<ILeftNavigationOption>) {
```

Comment:
Can you declare which type should this function return?

Reply:
I did it now